lib/fake_stripe.rb

```diff
@@ -30,9 +30,11 @@ def self.reset
   end
 
   def self.stub_stripe
-    Stripe.api_key = 'FAKE_STRIPE_API_KEY'
-    FakeStripe.reset
-    stub_request(:any, /api.stripe.com/).to_rack(FakeStripe::StubApp)
+    if defined?(Stripe)
```

Comment:
@gabebw You can use Stripe.JS without the `stripe` gem. For example, [Chargebee](https://apidocs.chargebee.com/docs/api/subscriptions#create_a_subscription) lets you create a subscription with the token from Stripe.js, letting you avoid having to use the `stripe` gem at all. :)